Makefile

```diff
@@ -21,7 +20,9 @@ build: build-web build-go
 build-go:
 	go build $(GO_BUILD_TAGS) ./...
 
-build-web: buf-web
+build-web:
+	npm ci --audit=false
```

Comment:
Do we still need to install/maintain any `npm` packages at the root level of the project? That single dependency was for how we previously generated `buf` stuff.

I imagine we can delete the `package.json` file at the root and just keep the one in `web/frontend`.

Reply:
great call -- yes the top-level dependency is only for generating protobufs, which this PR removes